packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -257,14 +266,49 @@ const TransactionHeader = memo(
           width={20}
           onSelect={e => dispatchSelected({ type: 'select-all', event: e })}
         />
-        <Cell value="Date" width={110} />
-        {showAccount && <Cell value="Account" width="flex" />}
-        <Cell value="Payee" width="flex" />
+        <Cell
+          value="Date"
+          width={110}
+          icon={field === 'date' && ascDesc}
+          onClick={e => onSortTable(e)}
```

Comment:
There isn’t any visible or accessible affordance for this — I think it would be better to show a button on hover (maybe ↕️?) and then have the icon on that button turn into a permanently visible up/down arrow when filtering is active on that column. (to be fair, I haven’t been able to test out the interaction since I can’t get it to work on the deploy preview)

Reply:
My thought was that hover pop-ups may be distracting and unneccessary. I opted for the "discoverable features" methodology here. Most users understand the idea of sorting columns by clicking on headers. Plus the default "Date" you suggested shows an up arrow on initial render so it hints people toward the fact that all the columns are sortable.

Thoughts?

Reply:
I think I'll wait for you to see it before I make any changes. 

Any idea why the preview isn't building properly?

Reply:
Preview seems to be fine, the end-to-end tests are failing because GitHub is once again broken 🤷

Tested again, and I like how it works over all! The icon could maybe stand to be a little bit smaller — right now it feels to me like it overpowers the column header text (although then it might get a little bit too thin if it was made smaller).

I like the idea of discoverable features (and think that it would definitely not make sense to show all of the buttons all the time. That said, I don’t think there are really any places in the UI where you are expected to click on a component that shows no signs of being interactive in order to perform an action. Wrapping the icon in a borderless button (like the Import button above it) would make it much clearer that you can click on it. Showing the buttons when hovering a column would also make it possible to know whether or not the column can be sorted without having to click.

Reply:
Wrapped it in the borderless button. I like the feel of it better. Thanks for the guidance.